spec/schema.json

```diff
@@ -65,6 +65,56 @@
         }
       }
     },
+    "/transactions/{transaction_id}/overseas-entity/{overseas_entity_id}": {
+      "put": {
+        "tags": [
+          "Overseas Entity"
+        ],
+        "summary": "Update an overseas entity registration submission",
+        "parameters": [
+          {
+            "name": "transaction_id",
+            "in": "path",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "overseas_entity_id",
+            "in": "path",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "requestBody": {
+          "content": {
+            "*/*": {
+              "schema": {
+                "$ref": "#/components/schemas/OverseasEntitySubmission"
+              }
+            }
+          },
+          "required": true
+        },
+        "responses": {
+          "201": {
```

Comment:
Probably it should be `200`, we are not creating a new resource

Reply:
True. Good spot. Will change.